test/Dialect/FIRRTL/emit-metadata.mlir

```diff
@@ -125,14 +125,14 @@ firrtl.circuit "top"
       firrtl.instance dut @dut()
       firrtl.instance mem1 @Mem1()
     }
-    firrtl.module @Mem1() {
+    firrtl.module private @Mem1() {
```

Comment:
Just out of curiosity: do we have to make all these modules private for the tests to work? Don't they just default to public which for the majority of tests is good enough?

Reply:
Yes, we assume that there is only one public module originating from the firrtl toplevel.